plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt

```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
```

Comment:
Origin scans are only for delta scans.

How about "apply exclusion rules to a non-delta scan"